src/articulation.js

```diff
@@ -198,6 +198,25 @@ export class Articulation extends Modifier {
     return true;
   }
 
+  static easyScoreHook({ articulations }, note, factory) {
```

Comment:
Just realized this isn't currently possible because the `Builder` calls the commit hooks and doesn't have access to the `EasyScore` instance.